File: src/main/java/gg/mineral/practice/commands/settings/TogglePartyRequestsCommand.java
package gg.mineral.practice.commands.settings;

import gg.mineral.practice.commands.PlayerCommand;
import gg.mineral.practice.managers.ProfileManager;
import gg.mineral.practice.util.messages.impl.ChatMessages;
import lombok.val;

public class TogglePartyRequestsCommand extends PlayerCommand {

    public TogglePartyRequestsCommand() {
        super("togglepartyrequests");
    }

    @Override
    public void execute(org.bukkit.entity.Player pl, String[] args) {
        val profile = ProfileManager.getOrCreateProfile(pl);
        profile.getRequestHandler().setPartyRequests(!profile.getRequestHandler().isPartyRequests());
        ChatMessages.PARTY_REQUESTS_TOGGLED.clone()
                .replace("%toggled%", profile.getRequestHandler().isPartyRequests() ? "enabled" : "disabled")
                .send(pl);
    }
}

File: src/main/kotlin/gg/mineral/practice/commands/settings/TogglePartyRequestsCommand.kt
package gg.mineral.practice.commands.settings

import dev.rollczi.litecommands.annotations.command.Command
import dev.rollczi.litecommands.annotations.context.Context
import dev.rollczi.litecommands.annotations.execute.Execute
import gg.mineral.practice.entity.Profile
import gg.mineral.practice.util.messages.impl.ChatMessages

@Command(name = "togglepartyrequests")
class TogglePartyRequestsCommand {
    @Execute
    fun execute(@Context profile: Profile) {
        profile.partyRequests = !profile.partyRequests
        profile.message(
            ChatMessages.PARTY_REQUESTS_TOGGLED.clone()
                .replace("%toggled%", if (profile.partyRequests) "enabled" else "disabled")
        )
    }
}
